python/tvm/topi/cuda/dense.py

```diff
@@ -77,8 +77,8 @@ def _callback(op):
 
 
 def _schedule_dense_small_batch(cfg, s, C):
-    A, _ = C.op.input_tensors
-    _, in_dim = get_const_tuple(A.shape)
+    A, weights = C.op.input_tensors
+    _, in_dim = get_const_tuple(weights.shape)
```

Comment:
Probably we can extent the logic a bit here to check both ```in_dim``` from data and weight. If both of them are symbolic, we can skip cfg define for ```in_dim``` since autotvm doesn't support dynamic shape kernel in general.

Reply:
Okay, I will update later.

Reply:
@kevinthesun updated